src/components/Card/Card.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
```

Comment:
we both use and don't use semi colons at the end of our lines - we should be consistent across the project

Reply:
On it.